Makefile

```diff
@@ -153,6 +153,7 @@ generate-dev: .prefix
 	NODE_ENV=development webpack --progress --colors --watch
 
 generate-mock: .prefix
+	go get -u github.com/groob/mockimpl
```

Comment:
If we're all on a recent-enough Go version (1.16 I think? Maybe 1.17?), this should probably be `go install github.com/groob/mockimpl@latest` so the binary is installed and can be executed.

Reply:
Yeah that's Go 1.16, it should be fine: https://go.dev/doc/go1.16#go-command